### ISSUE TITLE
[iOS13] Multiple modals topBar rendering bug

### ISSUE THREAD

**Issue by simonasdev** [wix/react-native-navigation#5879]
### Issue Description

Noticed an issue when showing multiple modals - top bar not being layout correctly which causes wrong position of top bar buttons and some white area just above the component view: https://drive.google.com/open?id=10BMAIR6b9mHCaX9rhilvaJWRTGbOhc4H.
Appears only when showing a second modal, while one modal is already open. Weirdly, this issue doesn't occur for all modals, as can be seen in the video above. Also, top bar layouts correctly after some jerking of the modal view (video @ 6-8secs, 21-24secs).

Any workarounds for this behaviour apart from not using multiple modals?

### Steps to Reproduce / Code Snippets / Screenshots
```js                              
    // show some modal
    Navigation.showModal({
      stack: {
        children: [{
          component: { name: 'SettingsModal' }
        }]
      }
    })
    // ...
    // then show some other modal from inside it
    Navigation.showModal({
      stack: {
        children: [{
          component: { name: 'SecurityModal' }
        }]
      }
    })
```

---
### Environment
* React Native Navigation version: 4.5.4
* React Native version: 0.61.5
* Platform(s) (iOS, Android, or both?): iOS
* Device info (Simulator/Device? OS version? Debug/Release?): Simulator and device, ios 13, debug and release

**Comment by guyca** [comment]
Hey @simonasdev 
Are you able to reproduce this in the playground app?

**Comment by simonasdev** [comment]
hey. No, I was unable to reproduce it in playground app. I should also point out that modalPresentationStyle must be 'pageSheet' (or 'default' in iOS 13). Top bar also has a left button with icon. 
While playing more in my app I've noticed that this issue goes away when there's a react-native's yellow box rendered, or when I wrap my content in a react-native-gesture-handler's flat list, or when I set topBar `visible: false` in `static options` and then `mergeOptions` `visible: true` in `constructor`. So no idea what is going, probably some native-layout-rendering-race-condition-fuckery :D but for now I'll live with the latter hack.
It would be great if there was a way to force some kind of native re-layouting from JS side.
